playwright-java/src/test/java/com/lambdatest/PlaywrightTestSingle.java

```diff
@@ -18,8 +18,9 @@ public static void main(String[] args) {
             capabilities.addProperty("browserVersion", "latest");
             ltOptions.addProperty("platform", "Windows 10");
             ltOptions.addProperty("name", "Playwright Test");
-            ltOptions.addProperty("build", "Playwrite Testing in Java");
+            ltOptions.addProperty("build", "Playwright Testing in Java");
             ltOptions.addProperty("user", user);
+            ltOptions.addProperty("playwrightClientVersion", "1.20.2");
```

Comment:
for java use 
`java.util.Scanner s = new java.util.Scanner(Runtime.getRuntime().exec("npx playwright --version").getInputStream()).useDelimiter("\\A");
    String playwrightClientVersion = s.hasNext() ? s.next() : "";
    System.out.println(playwrightClientVersion.split(" ")[1]);`